src/client/gui/lib/catalogue/zone_dropdown.dart

```diff
@@ -0,0 +1,55 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_riverpod/flutter_riverpod.dart';
+
+import '../dropdown.dart';
+import '../providers.dart';
+
+class ZoneDropdown extends ConsumerWidget {
+  final String value;
+  final ValueChanged<String?> onChanged;
+
+  const ZoneDropdown({
+    super.key,
+    required this.value,
+    required this.onChanged,
+  });
+
+  @override
+  Widget build(BuildContext context, WidgetRef ref) {
+    return Column(
+      crossAxisAlignment: CrossAxisAlignment.start,
+      children: [
+        Row(
+          children: [
+            const Text('Pseudo zone'),
+            const SizedBox(width: 4),
+            Tooltip(
+              message:
+                  'Pseudo zones simulate availability zones in public clouds\nand allow for testing resilience against real-world incidents',
+              child:
+                  Icon(Icons.info_outline, size: 16, color: Colors.blue[700]),
+            ),
+          ],
+        ),
+        const SizedBox(height: 8),
+        (() {
+          final zones = ref.watch(zonesProvider);
+          final availableZones = zones.where((z) => z.available).toList();
+          final hasAvailableZones = availableZones.isNotEmpty;
+
+          return Dropdown<String>(
+            value: value,
+            onChanged: onChanged,
+            items: {
+              'auto': 'Auto (assign automatically)',
```

Comment:
Perhaps if there are no zones available, we shouldn't show the auto option. WDYT?
```suggestion
              if (hasAvailableZones)
                'auto': 'Auto (assign automatically)',
```

Reply:
I have no strong opinions either way, but I don't want to show an empty dropdown either because that would look like a bug. So we could add an else condition that adds "No zones available", or just keep it as is IMO

Reply:
Yeah, I see your point. You can leave it as it is for now and perhaps discuss with design later. Another totally different approach could've been maybe to show all the zones, but the unavailable ones to be grayed-out and unselectable? Any way, that's an improvement that can come later.